socli/socli.py

```diff
@@ -111,6 +111,16 @@ def socli(query):
             questions = search.get_questions_for_query(query)
             res_url = questions[0][2]
             display_results(search.so_url + res_url)  # Returned URL is relative to SO homepage
+
+    except IndexError:
+        for i in range(8):
+            res_url=questions[i][2]
+            try:
+                display_results(res_url)
+            except IndexError:
+                continue
+            break
+
```

Comment:
This code should be:
```python
    except IndexError:
        if len(questions > 1):
            for i in range(1, len(questions)):
                res_url = questions[i][2]
                try:
                    display_results(res_url)
                except IndexError:
                    continue
                break
        printer.print_warning("No results found...")
```
So that the code wont error out with indexerror when there is results less than length of 10.

Reply:
I think you meant `len(questions) > 1` instead?

Reply:
ooopsie, updated 😄 Thanks for pointing it out @hedythedev

Reply:
Yeah this looks much more in place with the rest of the code, thanks @gautamkrishnar @ankushduacodes and @hedythedev